src/rules/at-forward-before-use/index.js

```diff
@@ -0,0 +1,39 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-forward-before-use");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Forward function should be used before use function"
```

Comment:
Are these called functions, at-rules, or something else in this context, what's the correct term for `@use` and `@forward`?

Reply:
Also, this should follow the stylelint message guideline:

messages.expected = `Expected something to be something`

or

messages.rejected = `Unexpected something`